source/CIOpponentPlanet.cpp

```diff
@@ -75,7 +78,7 @@ void OpponentPlanet::increaseLayerSize() {
 void OpponentPlanet::setMass(float mass) {
     _mass = mass;
     if (_opponentNode != nullptr) {
-        _opponentNode->setProgress(mass / _winPlanetMass, getColor());
+        _opponentNode->setProgress(mass / (_layerLockinTotal * _winPlanetLayers * PLANET_MASS_DELTA + INITIAL_PLANET_MASS), getColor());
```

Comment:
This doesn't really make much sense to me. Can we switch it to something that only has to do with layer progress. So something like (number of locked in layers) + (progress on current layer) / layers to win the game

Reply:
Mhm, I can make that change if you'd like, but it shouldn't change functionality much.

Reply:
Took a deeper look. We need to change networking if we want that data on the opponent planet. This is the best we can do by tonight.